src/tests/test25-dynamic-backend-address.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+. hitch_test.sh
+set +o errexit
+
+echo "hitch_test hitch-tls.org" > /tmp/hitch_hosts
```

Comment:
`mktemp -d` for directories, with #209 you can safely use your current directory.